src/main/java/com/uid2/shared/model/S3Key.java

```diff
@@ -0,0 +1,14 @@
+package com.uid2.shared.model;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
```

Comment:
I don't think we need setters for any of these

Reply:
don't we need to use setId, set... when we create a new s3 key?

Reply:
Will do, will make all parameters final